code/modules/admin/verbs/randomverbs.dm

```diff
@@ -664,16 +664,6 @@ Traitors and the like can also be revived with the previous role mostly intact.
 
 	return
 
-/client/proc/cmd_admin_attack_log(mob/M in mob_list)

```

Comment:
This verb was not being used anywhere and I decided to just cut it(since you will be able to access attack logs by the player panel now).